src/components/User/Assets/AssetRequests/components/AssetRequestRejectForm.vue

```diff
@@ -1,8 +1,16 @@
 <template>
   <div class="trrf">
-    <h2>
-      Reject {{ assetRequest.code }}
-      {{ assetRequest.type === 'create_asset' ? 'create' : 'update' }} request
+    <h2 v-if="assetRequest.type === 'create_asset'">
+      {{ "asset-request-reject-form.reject" | globalize({
```

Comment:
Why you use the same localization differently

Reply:
Take out this string from conditions

Reply:
done